_appmap/configuration.py

```diff
@@ -269,6 +273,24 @@ def _load_functions(self):
 
         self.package_functions.update(modules)
 
+    def _check_path_values(self):
+        for item in self._config["packages"]:
+            path: str = item.get("path")
+            if path and not self._check_path_value(path):
```

Comment:
This code should also raise an exception if `path` is `None`. This case is currently not handled well, and produces an ugly error message.